stories/Select.tsx

```diff
@@ -144,23 +144,17 @@ storiesOf('Organisms|Select Components', module)
             const getSuggestions = (): TSuggestion[] => {
                 const suggestions = SUGGESTIONS.filter(
                     (item: TSuggestion) =>
-                        item.name.toLocaleLowerCase() !==
-                            store.get('inputValue').toLocaleLowerCase() &&
-                        item.name
-                            .toLocaleLowerCase()
-                            .includes(store.get('inputValue').toLocaleLowerCase()) &&
```

Comment:
`item.name.toLocaleLowerCase().includes(store.get('inputValue').toLocaleLowerCase())` should have not been removed as well as `store.set` below. Filtering by input functionality doesn't work without them.

Reply:
It was buggy and it is just an example implementation, so I removed it. It is not the main point here, and you can think of the suggestions like in profession suggestions which are smart and not only letter based.